gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-edit.component.html

```diff
@@ -32,9 +32,65 @@ <h3>{{ entrypointName }}</h3>
         [supportedQos]="supportedQos"
         [formControlName]="entrypoint.type + '-qos'"
       ></gio-form-qos>
+
+      <mat-card *ngIf="supportDlq" class="api-entrypoints-v4-edit__form-container__dlq">
+        <h3>Dead Letter</h3>
+        <p>
+          Define an external storage where each unsuccessfully pushed message will be stored, and configure a replay strategy to push them
+          again. You can select either an entire endpoint group or a single endpoint within a group.
+        </p>
+        <gio-form-slide-toggle class="api-entrypoints-v4-edit__form-container__dlq__toggle">
+          <div>Enable Dead Letter</div>
+          <mat-slide-toggle gioFormSlideToggle formControlName="enabledDlq" aria-label="Enable Dead Letter toggle"></mat-slide-toggle>
+        </gio-form-slide-toggle>
+
+        <mat-card *ngIf="enabledDlq" class="api-entrypoints-v4-edit__form-container__dlq">
+          <h3>Create from existing endpoint</h3>
+          <ng-container *ngIf="dlqElements.length !== 0; else noElement">
+            <mat-form-field>
+              <mat-select
+                aria-label="Dead letter endpoint"
+                required
+                formControlName="dlqElement"
+                class="api-entrypoints-v4-edit__form-container__dlq__select"
+              >
+                <mat-select-trigger>
+                  <ng-container *ngTemplateOutlet="dlqTpl; context: { $implicit: form.get('dlqElement').value }"> </ng-container>
+                </mat-select-trigger>
+                <mat-optgroup *ngFor="let optgroup of dlqElements" [label]="optgroup.name">
+                  <mat-option *ngFor="let dlqElement of optgroup.elements" [value]="dlqElement">
+                    <ng-container *ngTemplateOutlet="dlqTpl; context: { $implicit: dlqElement }"> </ng-container>
+                  </mat-option>
+                </mat-optgroup>
+              </mat-select>
+
+              <!-- Template use to display a DLQ element, use many times in the mat-select above-->
+              <ng-template #dlqTpl let-dlqElement>
+                {{ dlqElement?.name || '' }}
```

Comment:
Do you want to display the dlqElement if it is undefined? If not, then you can add an *ngIf to a container around line 62.

Reply:
done